index.html

```diff
@@ -1,4 +1,18 @@
 <head>
   <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.7/jquery.min.js"></script>
   <script src="script.js"></script>
-</head>
\ No newline at end of file
+</head>
+<body>
+  <img class="pix" src="http://i.giphy.com/S1phUc5mmaZqM.gif">
+  <img class="pix" src="http://i.giphy.com/eGe59ekUJEll6.gif">
+  <img class="pix" src="http://i.giphy.com/l41lNT5u8hCI92nQc.gif">
+  <script type="text/javascript" charset="utf-8">
+    var els = document.getElementsByClassName("pix");
+    function handleClick(e) {
+      console.log(this);
+    }
+    for(i=0 ; i < els.length ; i++){
```

Comment:
Hm, actually, should we remove this stuff before merging and only keep it in `solution`?